### PR TITLE
Improving publishing documentation. Moved Origin Trial back to the RE…

### LINKED ISSUE #31: document how to host & publish exported Web game
à la https://aframe.io/docs/0.7.0/introduction/hosting-and-publishing.html and http://lovr.org/docs/Distribution

### PR DESCRIPTION
**Pull request by delapuente**
…ADME compatibility section.

Fix #31

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,6 +25,12 @@ If the user does not have supported headset, browser or device, the content will
 
 This asset works by utilizing Unity's WebGL platform support and therefore shares the same limitations. Because of this, mobile support is limited and may not work. See [Unity's WebGL browser compatibility](https://docs.unity3d.com/2018.1/Documentation/Manual/webgl-browsercompatibility.html).
 
+#### Chrome for Android
+
+For WebVR content to work with the regular release version of Chrome for Android, users will need to [manually enable WebVR](https://webvr.rocks/chrome_for_android#setup) from `chrome://flags`.
```
Comment (cvan) [inline README.md]: can you mention “Gamepad Extensions” too?

File: docs/publishing.md [hunk 2]
```diff
@@ -1,7 +1,43 @@
 # Publishing
 
-## Chrome for Android
+Publishing on the Internet is not different from publishing a blog or the landing page of your game. Actually, with a web-based game you could embed the game in your site, putting it just one-click away from your players! No stores, no approvals, no installations... [on compatible browsers](../README.md#compatibility), they just work.
 
-For WebVR content to work with the regular release version of Chrome for Android, users will need to [manually enable WebVR](https://webvr.rocks/chrome_for_android#setup) from `chrome://flags`.
+## Choosing a static web server
 
-Alternatively, sites can register for a [WebVR Origin Trial token](https://webvr.rocks/chrome_for_android#what_is_the_webvr_origin_trial) which will allows the API to be available for whitelisted origins.
+Unless you use your own machines, it is likely you want to use a free hosting service. The main limit you'll find with free-hosting is restrictions over file-size, type and bandwith. WebVR exports propietary formats for packaged asset and those can be heavy; from few megabytes to several hundreds.
```
Comment (cvan) [inline docs/publishing.md]: `is restrictions` -> `are restrictions`

Comment (cvan) [reply docs/publishing.md]: I’d change the independent clause after the semicolon to be in parentheses (e.g., )

Comment (cvan) [reply docs/publishing.md]: typo: `bandwidth`

Comment (cvan) [reply docs/publishing.md]: `packaged asset` to `packaged assets`

File: docs/publishing.md [hunk 3]
```diff
@@ -1,7 +1,43 @@
...
+
+This document assumes you have already exported your game with Unity, targeting WebGL and using the WebVR template that comes with the assets. To know how to export your project for WebVR, please refer to [Setting up a Unity project for WebVR](./project-setup.md).
```
Comment (cvan) [inline docs/publishing.md]: `with the assets` -> `with the assets package`?

File: docs/publishing.md [hunk 4]
```diff
@@ -1,7 +1,43 @@
...
+
+### Deploying with `surge`
+
+The most simple way to publish your game under a custom domain is using `surge`. Install it [using `npm`](https://nodejs.org/en/download/):
```
Comment (cvan) [inline docs/publishing.md]: `most simple` -> `simplest`

File: docs/publishing.md [hunk 5]
```diff
@@ -1,7 +1,43 @@
...
+
+```
+$ npm install --global surge
```
Comment (cvan) [inline docs/publishing.md]: you can remove the `$` (for easy copy and pasting)

File: docs/publishing.md [hunk 6]
```diff
@@ -1,7 +1,43 @@
...
-Alternatively, sites can register for a [WebVR Origin Trial token](https://webvr.rocks/chrome_for_android#what_is_the_webvr_origin_trial) which will allows the API to be available for whitelisted origins.
+Unless you use your own machines, it is likely you want to use a free hosting service. The main limit you'll find with free-hosting is restrictions over file-size, type and bandwith. WebVR exports propietary formats for packaged asset and those can be heavy; from few megabytes to several hundreds.
+
+This document assumes you have already exported your game with Unity, targeting WebGL and using the WebVR template that comes with the assets. To know how to export your project for WebVR, please refer to [Setting up a Unity project for WebVR](./project-setup.md).
+
+### Deploying with `surge`
+
+The most simple way to publish your game under a custom domain is using `surge`. Install it [using `npm`](https://nodejs.org/en/download/):
+
+```
```
Comment (cvan) [inline docs/publishing.md]:
```sh

File: docs/publishing.md [hunk 7]
```diff
@@ -1,7 +1,43 @@
...
+$ npm install --global surge
+```
+
+And inside your build directory, simply run:
+
+```
```
Comment (cvan) [inline docs/publishing.md]: same

File: docs/publishing.md [hunk 8]
```diff
@@ -1,7 +1,43 @@
...
+$ surge
+```
```
Comment (cvan) [inline docs/publishing.md]: same

File: docs/publishing.md [hunk 9]
```diff
@@ -1,7 +1,43 @@
...
+
+It's that easy. **Seriously**.
```
Comment (cvan) [inline docs/publishing.md]: feel free to include the `.` in the bolded word

File: docs/publishing.md [hunk 10]
```diff
@@ -1,7 +1,43 @@
...
+
+### Deploying with `surge`
+
+The most simple way to publish your game under a custom domain is using `surge`. Install it [using `npm`](https://nodejs.org/en/download/):
+
+```
+$ npm install --global surge
+```
+
+And inside your build directory, simply run:
```
Comment (cvan) [inline docs/publishing.md]: perhaps we should note the root directory, not the `Build/` directory of the package files?

File: docs/publishing.md [hunk 11]
```diff
@@ -1,7 +1,43 @@
...
+
+```
+$ surge
+```
+
+It's that easy. **Seriously**.
+
+![Using surge](https://surge.sh/images/help/getting-started-with-surge.gif)
+
+Refer to [`surge` documentation](https://surge.sh/help/getting-started-with-surge) to learn how to futher customize the deploy.
```
Comment (cvan) [inline docs/publishing.md]: ` to` -> ` to the`

File: docs/publishing.md [hunk 12]
```diff
@@ -1,7 +1,43 @@
...
+
+Remember `surge` will serve your files under both `http` and `https` connections but the free plan won't redirect you from `http` to `https` automatically. Remember to share the `https` version of your site!
```
Comment (cvan) [inline docs/publishing.md]: enhancement: perhaps we can include a simple JS snippet that redirects to `https` if `window.location.protocol !== 'https:' && window.isSecureContext === true`

File: docs/publishing.md [hunk 13]
```diff
@@ -1,7 +1,43 @@
...
+
+![Using surge](https://surge.sh/images/help/getting-started-with-surge.gif)
+
+Refer to [`surge` documentation](https://surge.sh/help/getting-started-with-surge) to learn how to futher customize the deploy.
+
+Remember `surge` will serve your files under both `http` and `https` connections but the free plan won't redirect you from `http` to `https` automatically. Remember to share the `https` version of your site!  
+
+### Deploying on GitHub
+
+Another good choice is using GitHub Pages. If your build folder [meets the constraints](https://help.github.com/articles/what-is-github-pages/#usage-limits), you can convert it into a `git` repository and publish it. Refer to [Create a repo](https://help.github.com/articles/create-a-repo/) and [Configuring a publishing source for GitHub Pages](https://help.github.com/articles/configuring-a-publishing-source-for-github-pages/#enabling-github-pages-to-publish-your-site-from-master-or-gh-pages) to find out how to create and publish your game.
```
Comment (cvan) [inline docs/publishing.md]: for the first mention of `GitHub Pages`, can you link to https://pages.github.com/?

File: docs/publishing.md [hunk 14]
```diff
@@ -1,7 +1,43 @@
...
+
+### Reaching bandwith restrictions
```
Comment (cvan) [inline docs/publishing.md]: typo: `bandwidth`

Comment (cvan) [reply docs/publishing.md]: `Reaching` -> `Exceeding`? or remove the first word altogether?

File: docs/publishing.md [hunk 15]
```diff
@@ -1,7 +1,43 @@
...
+
+Even if your files are not that big, we hope you'll be successful and attract tons of users to your game. That means potentially thousands of gigabytes being downloaded per month. We are working on improving the template to [save as much bandwith as possible](https://github.com/mozilla/unity-webvr-export/issues/98) but eventually, you'll reach bandwith limits. If that's the case, consider using a CDN or get a paid hosting plan. There are [very good and inexpensive plans out there](http://www.turiyaware.com/finding-the-right-hosting-for-indie-game-developers/).
```
Comment (cvan) [inline docs/publishing.md]: we ought to ask GitHub to conditionally gzip for `.unityweb` packages

Comment (cvan) [reply docs/publishing.md]: good find for that link for hosting options

File: docs/publishing.md [hunk 16]
```diff
@@ -1,7 +1,43 @@
...
+
+#### Integrating with a CDN
+
+Refer to the section `Moving build output files` from Unity's [WebGL building documentation](https://docs.unity3d.com/Manual/webgl-building.html) to learn how to configure the template to work with an external CDN.
```
Comment (cvan) [inline docs/publishing.md]: add a newline at the EOF

Comment (cvan) [reply docs/publishing.md]: perhaps we should name some common CDNs in the first paragraph

File: docs/publishing.md (hunk 4 above)
Comment (delapuente) [reply docs/publishing.md]: Oh, man, I always mistook this.

**Pull request merged**